README.md

```diff
@@ -2,369 +2,6 @@
-```python
-addresses = set(['address1','address2'])
-async for address, tx in client.monitor_accounts_payments(addresses):
-   ...
-```
-
-## Channels
-
-One of the most sensitive points in Stellar is [transaction sequence](https://www.stellar.org/developers/guides/concepts/transactions.html#sequence-number).
-In order for a transaction to be submitted successfully, this number should be correct. However, if you have several 
-SDK instances, each working with the same wallet account or channel accounts, sequence collisions will occur. 
- 
-We highly recommend to keep only one KinAccount instance in your application, having unique channel accounts.
-Depending on the nature of your application, here are our recommendations:
-
-1. You have a simple (command line) script that sends transactions on demand or only once in a while. 
-In this case, the SDK can be instantiated with only the wallet key, the channel accounts are not necessary.
-
-2. You have a single application server that should handle a stream of concurrent transactions. In this case, 
-you need to make sure that only a single instance of a KinAccount initialized with multiple channel accounts.
-
-3. You have a number of load-balanced application servers. Here, each application server should a) have the setup outlined
-above, and b) have its own channel accounts. This way, you ensure you will not have any collisions in your transaction
-sequences.
-
-### Creating Channels
-The kin sdk allows you to create HD (highly deterministic) channels based on your seed and a passphrase to be used as a salt.
-As long as you use the same seed and passphrase, you will always get the same seeds.
-
-```
-import kin.utils
-
-channels = utils.create_channels(master_seed, environment, amount, starting_balance, salt)
-
-# "channels" will be a list of seeds the sdk created for you, that can be used when initializing the KinAccount object.
-```
-If you just wish to get the list of the channels generated from your seed + passphrase combination without creating them
-```python
-channels = utils.get_hd_channels(master_seed, salt, amount)
-```
-
-
-## License
-The code is currently released under [Kin Ecosystem SDK License](LICENSE.pdf).
-
-
-## Contributing
-See [CONTRIBUTING.md](CONTRIBUTING.md) for SDK contributing guidelines. 
-
+The Kin SDK for Python is meant to be used as a back-end service. It can perform actions for your client apps (iOS, Android, etc.) and also operate as a server for you to build services on top of the Kin blockchain. The SDK can, for example, take care of communicating with the Kin Blockchain on behalf of the client to create accounts and whitelist transactions. It can also monitor blockchain transactions so that you can implement broader services. It's up to you how to integrate the SDK in your overall architecture and managing server up-time.
 
+For full documentation you can refer to our website [Kin Python SDK docs](https://docs.kin.org/python/sdk)
```

Comment:
use this nice image I've put in [node readme](https://github.com/kinecosystem/kin-sdk-node) (I've linked direclty to partners site which is bad, but the image is nice)